ggml-cuda.cu

```diff
@@ -2689,6 +2746,12 @@ void ggml_init_cublas() {
     static bool initialized = false;
 
     if (!initialized) {
+
+#ifdef __HIP_PLATFORM_AMD__
+        rocblas_initialize();
+        CUDA_CHECK(cudaDeviceSynchronize());
+#endif
+
```

Comment:
Could you add a comment saying that it's a fix for an upstream bug in rocblas, with maybe a link to the rocblas issue: https://github.com/ROCmSoftwarePlatform/rocBLAS/issues/1346. 

Currently, this change is hard to understand without knowing of this issue.